shared/misc/galaxy_ansible_role_meta_template.yml

```diff
@@ -0,0 +1,30 @@
+galaxy_info:
+  author: SCAP Security Guide
+  description: @DESCRIPTION@
+  company: OpenSCAP
+  
+  issue_tracker_url: https://github.com/OpenSCAP/scap-security-guide/issues
+
+  license: CC0
+
+  min_ansible_version: 1.2
+  
+  platforms:
+  - name: GenericBSD
+    versions:
+    - any
+  - name: GenericLinux
+    versions:
+    - any
+  - name: GenericUNIX
+    versions:
+    - any
+  
+  galaxy_tags:
+  - hardening
+  - openscap
+  - ssg
+  - security
+  - complaince
```

Comment:
Should our major profiles be added here?
- stig
- usgcb
- fisma
- pci
- fedramp
- c2s

Reply:
hmm, that may make search results worse on ansible galaxy. right now if you search for these you will always find only the roles generated form these profiles. I think if we add these here you will get all our roles and it may be harder to find the specific role you are looking for.

Reply:
Maybe the profile tag could be templated like description, so that we have only relevant tags.